server/server.go

```diff
@@ -744,27 +744,14 @@ func (s *Server) GetConfig() *config.Config {
 	for i, sche := range sches {
 		payload[sche] = configs[i]
 	}
-	cfg.Schedule.SchedulersPayload = payload
+	cfg.SchedulersPayload = payload
 	return cfg
 }
 
 // GetScheduleConfig gets the balance config information.
 func (s *Server) GetScheduleConfig() *config.ScheduleConfig {
 	cfg := &config.ScheduleConfig{}
 	*cfg = *s.scheduleOpt.Load()
-	storage := s.GetStorage()
```

Comment:
Why change this? as you see,  the payload do not in `scheduleConfig`, but we want to display it.

Reply:
PTAL

Reply:
I move the `SchedulersPayload`  from `scheduleConfig` to `Config`.

Reply:
The `schedulers-payload` can be got when execute `pd-ctl config show all`.

Reply:
got it, thanks.